File: help/pt_pt/exercicios_da_ficha_11/Ficha11_007/Ficha11_007.cpp
// Programador           -

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha11_007 {

    public:
        int main() {
            linha(3 ) ;
            linha(5 ) ;
            linha(7 ) ;
            linha(5 ) ;
            linha(3 ) ;
            return 0;
        }

        //::::::::::::::::::::::::::::::::::::::::::::::::::
        //User Defined Functions Code
        //::::::::::::::::::::::::::::::::::::::::::::::::::
        void linha( int n ){
            int i  = 0 ;
            for( int i = 0; i < n; i = i + 1 ) {
                cout << "*";
            }
            cout << "\n";
        }
};// end of class Ficha11_007
// Execute main function
int main(){
    Ficha11_007 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:09

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_007/Ficha11_007.cs
// Programador           -

using System.IO;
using System;
public class Ficha11_007 {

    //Main Function
    static void Main() {
        linha(3 ) ;
        linha(5 ) ;
        linha(7 ) ;
        linha(5 ) ;
        linha(3 ) ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha( int n){
        for( int i = 0; i < n; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:09
